### PR TITLE
Pangram approaches

### PR DESCRIPTION
**Pull request by bobahop**
Initial commit for file structure and code. Explanatory details to follow...

### CODE REVIEW

**Comment by bobahop** [comment]
@ErikSchierboom Rust Pangram approaches ready for review

File: exercises/practice/pangram/.approaches/bitfield/snippet.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+    for letter in sentence.chars() {
+        if letter >= 'a' && letter <= 'z' {
+            letter_flags |= 1 << (letter as u8 - A_LCASE);
+        } else if letter >= 'A' && letter <= 'Z' {
+            letter_flags |= 1 << (letter as u8 - A_UCASE);
+        }
+    }
+    letter_flags == ALL_26_BITS_SET
```
Comment (ErikSchierboom) [inline exercises/practice/pangram/.approaches/bitfield/snippet.txt:8]:
```suggestion
for letter in sentence.chars() {
    if letter >= 'a' && letter <= 'z' {
        letter_flags |= 1 << (letter as u8 - A_LCASE);
    } else if letter >= 'A' && letter <= 'Z' {
        letter_flags |= 1 << (letter as u8 - A_UCASE);
    }
}
letter_flags == ALL_26_BITS_SET
```

**Review by ErikSchierboom — APPROVED** [review]
Tiny nit

File: exercises/practice/pangram/.approaches/bitfield/snippet.txt (hunk 1 above)
Comment (bobahop) [reply exercises/practice/pangram/.approaches/bitfield/snippet.txt:8]: It seems no matter how conscientious I try to be about the indents they still get past me sometimes.

Comment (ErikSchierboom) [reply exercises/practice/pangram/.approaches/bitfield/snippet.txt]: No worries, now that GitHub suggestions support me using shift+tab, it is very easy to suggest the right indentation.

**Comment by petertseng** [comment]
Again: When squashing to merge, please edit the commit message to be readable in instances where the default is not.

**Pull request merged**